lib/lhm/version.rb

```diff
@@ -2,5 +2,5 @@
 # Schmidt
 
 module Lhm
-  VERSION = '3.1.0'
+  VERSION = '3.1.1'
```

Comment:
@berlincount I try to bump versions in isolated commits. Makes it easier to identify what changed between specific releases. Also, you need to include `Gemfile.lock` changes that accompany this change. See https://github.com/Shopify/lhm#releasing-new-versions